pkg/local_object_storage/engine/inhume.go

```diff
@@ -241,6 +252,64 @@ func (e *StorageEngine) processAddrDelete(addr oid.Address, deleteFunc func(*sha
 	return retErr
 }
 
+func (e *StorageEngine) processSplitObjectWithNoLink(addr oid.Address, si *objectSDK.SplitInfo) []oid.Address {
+	var children []oid.Address
+	e.log.Info("root object has no link object in split upload",
+		zap.Stringer("addrBeingInhumed", addr))
+
+	var (
+		fs        objectSDK.SearchFilters
+		newCursor []byte
+		count     = uint16(10)
```

Comment:
Doesn't make much sense limiting to 10 here, we need all elements anyway so you can use 1000 which is a normal search limit.